Detectors/GlobalTracking/src/MatchTOF.cxx

```diff
@@ -501,6 +501,9 @@ bool MatchTOF::loadTracksNextChunk()
   ///< load next chunk of tracks to be matched to TOF
   while (++mCurrTracksTreeEntry < mInputTreeTracks->GetEntries()) {
     mInputTreeTracks->GetEntry(mCurrTracksTreeEntry);
+    if (mTracksArrayInp)
+      delete mTracksArrayInp;
```

Comment:
never delete the span, it is not supposed to manage the memory it points to. Once you change it from pointer to the object, just do here:
````
mInputTreeTracks->GetEntry(mCurrTracksTreeEntry);
mTracksArrayInp = gsl::span<const o2::dataformats::TrackTPCITS>{mTracksArrayInpVect.data(),mTracksArrayInpVect.size()};
````

Then access it as usual vector, i.e. ``mTracksArrayInp.size``, ``mTracksArrayInp[i]`` etc.

Reply:
removed (now also object instead of pointer)